acros.aux

```diff
@@ -0,0 +1,23 @@
+\relax 
```

Comment:
aux files should not be version controlled (they are generated by latex from the .tex files). This should be removed from the PR.